iphone/Classes/TiUIWebView.m

```diff
@@ -322,6 +322,17 @@ -(id)url
 	return url;
 }
 
+-(id)setAllowsLinkPreview_:(id)value
+{
+#if IS_XCODE_7
+    if ([TiUtils isIOS9OrGreater] == NO) {
+        return;
+    }
+    ENSURE_TYPE(value, NSNumber);
+    [webview setAllowsLinkPreview:[TiUtils boolValue:value]];
+#endif
+}
+
```

Comment:
Didn't crash on the iPhone 6S test device, but of course it needs to be `void`. Fixed.